<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowIncome</class>
 <widget class="QMainWindow" name="WindowIncome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>養生館櫃台結帳</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget_income">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_requery"/>
   <addaction name="separator"/>
   <addaction name="action_export_daily_excel"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_export_daily_excel"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.svg</normaloff>../icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>關閉視窗</string>
   </property>
  </action>
  <action name="action_requery">
   <property name="icon">
    <iconset>
     <normaloff>../icons/gtk-refresh.svg</normaloff>../icons/gtk-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>重新查詢</string>
   </property>
   <property name="toolTip">
    <string>重新查詢</string>
   </property>
  </action>
  <action name="action_export_daily_excel">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-export.svg</normaloff>../icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>匯出 excel</string>
   </property>
   <property name="toolTip">
    <string>匯出日報表 excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
